### PR TITLE
Eric issue 03

### PR DESCRIPTION
**Pull request by eric-song1773**
- Created a mockup page of Hearing List

### CODE REVIEW

File: app/imports/ui/pages/HearingList.jsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { MDBTable, MDBTableHead, MDBTableBody, MDBPagination, MDBPaginationLink, MDBPaginationItem } from 'mdb-react-ui-kit';
+// import { PAGE_IDS } from '../utilities/PageIDs';
+
+/* A simple static component to render some text for the landing page. */
+const TestimonyList = () => (
```
Comment (William-Liang808) [inline app/imports/ui/pages/HearingList.jsx:7]: File is HearingList but your static component is named TestimonyList

File: app/imports/ui/pages/HearingList.jsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        <MDBPaginationLink href="#">3</MDBPaginationLink>
+      </MDBPaginationItem>
+      <MDBPaginationItem>
+        <MDBPaginationLink href="#">Next</MDBPaginationLink>
+      </MDBPaginationItem>
+    </MDBPagination>
+  </Container>
+);
+
+export default TestimonyList;
```
Comment (William-Liang808) [inline app/imports/ui/pages/HearingList.jsx:180]: Same down here

**Review by William-Liang808 — CHANGES_REQUESTED** [review]
Should be good after rename

**Review by William-Liang808 — APPROVED** [review]
looks good

**Review by zakgilbert — APPROVED** [review]

**Pull request merged**